frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.tsx

```diff
@@ -0,0 +1,86 @@
+import { c, t } from "ttag";
+
+import { useSearchQuery } from "metabase/api";
+import { BROWSE_MODELS_SEARCH_REQUEST } from "metabase/browse/constants";
+import { useUserSetting } from "metabase/common/hooks";
+import CollapseSection from "metabase/components/CollapseSection";
+import { DelayedLoadingAndErrorWrapper } from "metabase/components/LoadingAndErrorWrapper/DelayedLoadingAndErrorWrapper";
+import CS from "metabase/css/core/index.css";
+import { Flex, Loader } from "metabase/ui";
+
+import { PaddedSidebarLink, SidebarHeading } from "../MainNavbar.styled";
+import type { SelectedItem } from "../types";
+
+export const BrowseNavSection = ({
+  nonEntityItem,
+  onItemSelect,
+  hasDataAccess,
+}: {
+  nonEntityItem: SelectedItem;
+  onItemSelect: () => void;
+  hasDataAccess: boolean;
+}) => {
+  const BROWSE_MODELS_URL = "/browse/models";
+  const BROWSE_DATA_URL = "/browse/databases";
+
+  const modelsResult = useSearchQuery({
+    ...BROWSE_MODELS_SEARCH_REQUEST,
+    model_ancestors: false,
+  });
+  const noModelsExist = modelsResult.data?.data.length === 0;
```

Comment:
We don't have a better endpoint for checking the existence of models at the moment 😕